src/main/resources/static/css/app-ie8.css

```diff
@@ -0,0 +1,79 @@
+#header-container {
```

Comment:
All of the static content was pinched out of the Auth server without applying much thought...